src/Mutation/Mutation.php

```diff
@@ -62,6 +63,10 @@ class Mutation
     private $originalFileAst;
     private $tests;
     private $coveredByTests;
+    /**
+     * @var float|null
+     */
+    private $nominalTimeToTest;
```

Comment:
What is the unit for this time, seconds, milliseconds, etc? 

Suggest:
1. update name to `nominalTimeInSecondsToTest` or whatever unit actually is.
2. or better still create a value object `Seconds`, 'MiliSeconds` to aid clarity here.

Reply:
It's all seconds here.